lib/factory_bot/attribute_assigner.rb

```diff
@@ -31,7 +31,10 @@ def hash
     private
 
     def method_tracking_evaluator
-      @method_tracking_evaluator ||= Decorator::AttributeHash.new(decorated_evaluator, attribute_names_to_assign)
+      @method_tracking_evaluator ||= Decorator::AttributeHash.new(
```

Comment:
This change is not strictly related, but this line was long and I happened to be looking here as I was following the page through to the AttributeHash